### ISSUE TITLE
Error occurred with autocorrect

### ISSUE THREAD

**Issue by dlecocq** [rubocop/rubocop#512]
The error message mentioned that I should open an issue. In particular, when running

``` bash
git clone https://github.com/seomoz/qless.git
cd qless
git checkout 9e5ef9cae19ef8ba59d0ac51636bf068aa0ff9a1
rubocop -a lib
```

I get:

```
3 errors occurred:
An error occurred while AndOr cop was inspecting /Users/dan/SEOmoz/qless/qless/lib/qless/queue.rb.
An error occurred while AndOr cop was inspecting /Users/dan/SEOmoz/qless/qless/lib/qless/queue.rb.
An error occurred while AndOr cop was inspecting /Users/dan/SEOmoz/qless/qless/lib/qless/server.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.13.1 (using Parser 2.0.0.pre8, running on ruby 1.9.3 x86_64-darwin11.2.0)
```

I hope this helps!
